lib/ClangImporter/ClangImporter.cpp

```diff
@@ -7446,6 +7446,31 @@ static bool hasImportAsRefAttr(const clang::RecordDecl *decl) {
          });
 }
 
+// TODO: Move all these utility functions in a new file ClangImporterUtils.h
+// rdar://138803759
+static bool hasImmortalAtts(const clang::RecordDecl *decl) {
```

Comment:
nit: spell out "attributes" abbreviation to be consistent with other functions

```suggestion
static bool hasImmortalAttrs(const clang::RecordDecl *decl) {
```

Reply:
Oops. I'll fix in a follow up refactoring patch. Thanks!
Ref: rdar://138803759